genome_sampler/plugin_setup.py

```diff
@@ -341,6 +344,37 @@ def _4(fmt: GISAIDDNAFASTAFormat) -> DNASequencesDirectoryFormat:
     description='Combine list of id selections into single id selection.'
 )
 
+
+plugin.methods.register_function(
+    function=label_seqs,
+    inputs={'seqs': FeatureData[AlignedSequence]},
+    parameters={
+        'delimiter': Str % Choices('|', ',', '+', ':', ';'),
+        'metadata': Metadata,
+        'columns': List[Str],
+    },
+    outputs=[('labeled_seqs', FeatureData[AlignedSequence])],
+    input_descriptions={'seqs': 'The sequences to be re-labeled.'},
+    parameter_descriptions={
+        'delimiter': 'The delimiter between the sequence id and each metadata'
+                     ' entry.',
+        'metadata': 'The metadata to embed in the header.',
+        'columns': 'The columns in the metadata to be used.',
+    },
+    output_descriptions={
+        'labeled_seqs': 'The re-labeled sequences.'
+    },
+    name='Re-label sequences',
+    description='Modifies sequence identifiers either by adding or removing'
+                ' metadata. If metadata and one or more columns are provided,'
+                ' the specified metadata columns will be added to the sequence'
+                ' id following the original sequence id and separated by'
+                ' delimiter. If metadata and columns are not provided, the'
```

Comment:
_delimiter_ should be in tick marks here, right? I think the CLI renders it differently if so, since it's a parameter name.

Reply:
good catch!